packages/jazz-tools/src/svelte/jazz.class.svelte.ts

```diff
@@ -306,3 +307,18 @@ export class SyncConnectionStatus {
     return this.#ctx.current?.connected() ?? false;
   }
 }
+
+function getResolveQuery(
+  Schema: CoValueClassOrSchema,
+  // We don't need type validation here, since this is an internal API
+  resolveQuery?: ResolveQuery<any>,
+): ResolveQuery<any> {
+  if (resolveQuery) {
+    return resolveQuery;
+  }
+  // Check the schema is a CoValue schema (and not a CoValue class)
+  if ("resolve" in Schema) {
+    return Schema.resolve;
+  }
+  return true;
+}
```

Comment:
Could you add a test for this?

I can help with the setup.

Reply:
Had to split the tests into a Svelte component test (otherwise it's impossible to test the classes' behavior) and a type test: [`8f6b7e4` (#3147)](https://github.com/garden-co/jazz/pull/3147/commits/8f6b7e4a047e3f145a2762374b538ec262152b75)